<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1688709539168' stop='1688709543325'>
    <name>login:</name>
    <title>login:</title>
    <test-cases>
        <test-case start='1688709540078' status='failed' stop='1688709541549'>
            <name>user login | {"username":"standard_user","password":"secret_sauce"} @smoke1</name>
            <title>user login | {"username":"standard_user","password":"secret_sauce"} @smoke1</title>
            <labels>
                <label name='language' value='javascript'/>
                <label name='framework' value='codeceptjs'/>
                <label name='tag' value='@smoke1'/>
            </labels>
            <parameters/>
            <steps>
                <step start='1688709540630' status='passed' stop='1688709541548'>
                    <name>I am on page "/"</name>
                    <title>I am on page "/"</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments>
                <attachment title='Main session - Last Seen Screenshot' source='64c2acb7-901c-4c7f-a76f-6a419a475d77-attachment.png' type='image/png' size='30086'/>
            </attachments>
            <failure>
                <message>user_data.fillLoginFields is not a function</message>
                <stack-trace>TypeError: user_data.fillLoginFields is not a function
                    at Test.&lt;anonymous> (C:\POC\tests\login_test.js:19:15)
                    at test.fn (C:\POC\node_modules\codeceptjs\lib\scenario.js:92:14)
                    at callFnAsync (C:\POC\node_modules\mocha\lib\runnable.js:394:21)
                    at Runnable.run (C:\POC\node_modules\mocha\lib\runnable.js:338:7)
                    at Runner.runTest (C:\POC\node_modules\mocha\lib\runner.js:666:10)
                    at C:\POC\node_modules\mocha\lib\runner.js:789:12
                    at next (C:\POC\node_modules\mocha\lib\runner.js:581:14)
                    at C:\POC\node_modules\mocha\lib\runner.js:591:7
                    at next (C:\POC\node_modules\mocha\lib\runner.js:474:14)
                    at cbHookRun (C:\POC\node_modules\mocha\lib\runner.js:539:7)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1688709541775' status='failed' stop='1688709543126'>
            <name>user login | {"username":"locked_out_user","password":"secret_sauce"} @smoke1</name>
            <title>user login | {"username":"locked_out_user","password":"secret_sauce"} @smoke1</title>
            <labels>
                <label name='language' value='javascript'/>
                <label name='framework' value='codeceptjs'/>
                <label name='tag' value='@smoke1'/>
            </labels>
            <parameters/>
            <steps>
                <step start='1688709542277' status='passed' stop='1688709543126'>
                    <name>I am on page "/"</name>
                    <title>I am on page "/"</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments>
                <attachment title='Main session - Last Seen Screenshot' source='10feaf6d-e17a-41c1-bc14-c1355c956368-attachment.png' type='image/png' size='30086'/>
            </attachments>
            <failure>
                <message>user_data.fillLoginFields is not a function</message>
                <stack-trace>TypeError: user_data.fillLoginFields is not a function
                    at Test.&lt;anonymous> (C:\POC\tests\login_test.js:19:15)
                    at test.fn (C:\POC\node_modules\codeceptjs\lib\scenario.js:92:14)
                    at callFnAsync (C:\POC\node_modules\mocha\lib\runnable.js:394:21)
                    at Runnable.run (C:\POC\node_modules\mocha\lib\runnable.js:338:7)
                    at Runner.runTest (C:\POC\node_modules\mocha\lib\runner.js:666:10)
                    at C:\POC\node_modules\mocha\lib\runner.js:789:12
                    at next (C:\POC\node_modules\mocha\lib\runner.js:581:14)
                    at C:\POC\node_modules\mocha\lib\runner.js:591:7
                    at next (C:\POC\node_modules\mocha\lib\runner.js:474:14)
                    at cbHookRun (C:\POC\node_modules\mocha\lib\runner.js:539:7)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>